### DISCUSSION TITLE
Incorrect API Documentation for methods without parameters in "macroable" classes

### DISCUSSION THREAD

**Discussion by josephmiclaus** [laravel/framework#46976] [General]
I was going through the Laravel API Documentation and it seems that methods without parameters in "macroable" classes aren't generated correctly.

The method description's first letter is replaced with $ and the Parameters section looks wrong.

Example: https://laravel.com/api/10.x/Illuminate/Database/Schema/ColumnDefinition.html#method_autoIncrement

EDIT: Same goes for 9.x and 8.x documentations

**Comment by eapacheco** [comment]
The API documentation is not implemented by Laravel community, but by [Doctum, a PHP API documentation generator. Fork of Sami](https://github.com/code-lts/doctum). Refer to their GitHub issues and/or PRs!

**Reply by josephmiclaus** [reply]
Thanks, but I'm not currently looking to solving this issue myself. I only wanted to raise attention to the provided documentation at [laravel.com/api](https://laravel.com/api).
